### ISSUE TITLE
tanso.net Doesn’t load

### ISSUE THREAD

**Issue by pitr** [pitr/gemini-ios#17]
gemini://tanso.net

```
The operation couldn’t be completed. (OSStatus error -9806.)
```

**Comment by j4lc** [comment]
I think TLS should be upgraded.
See https://github.com/a-h/gemini/issues/6

**Comment by pitr** [comment]
fixed in d7162b7c, will test locally for a bit and push to the App Store

**Comment by pitr** [comment]
latest version with the fix is in review so it's in Apple's hands, closing this ticket
